src/components/Combobox/ComboboxDropdownListItem.js

```diff
@@ -41,6 +41,7 @@ const ComboboxDropdownListItem = (props) => {
   const itemClasses = [
     'slds-listbox__option',
     'slds-listbox__option_plain',
+    'slds-listbox__option-heade',
```

Comment:
`slds-listbox__option-heade`

Reply:
Shouldn't be there in the first place :see_no_evil: Thanks for pointing this out